docs/reference/search/search-template.asciidoc

```diff
@@ -415,49 +427,66 @@ POST /_search/template/<templatename>
     }
 }
 ------------------------------------------
+// CONSOLE
+// TEST[continued]
+
+//////////////////////////
+
```

Comment:
Probably worth adding a short comment. At this point we mostly understand why we have these but it is nice to have the comment anyway.